src/clients/node/README.md

```diff
@@ -280,6 +290,14 @@ for (let i = 0; i < transfers.length; i += BATCH_SIZE) {
 }
 ```
 
+### Queues and Workers
+
+If you are making requests to TigerBeetle from workers
+pulling jobs from a queue, you can batch requests to
+TigerBeetle by having the worker act on multiple jobs from
+the queue at once rather than one at a time. i.e. pulling
+multiple jobs from the queue rather than just one.
```

Comment:
@sentientwaffle calling attention to new sections for queues and workers.

I think we should also put this in other global docs but since this section (expand to see above it) is about Batching, might as well add here now.

Reply:
:+1:  (https://docs.tigerbeetle.com/usage/integration#batching-events)

Reply:
Nice! Should I just add the same section to the end of that doc, basically?

Reply:
Yes, perfect :+1:

Reply:
Done, thanks.